.github/workflows/testing.yml

```diff
@@ -20,6 +20,11 @@ jobs:
       matrix:
         include:
 
+        - name: Python 3.11
+          os: ubuntu-latest
+          python: '3.11'
+          toxenv: py311
```

Comment:
We'll need to go to branch protection rules on the GitHub repository for `main` to update the names of required tests.